js/utilities.js

```diff
@@ -731,6 +731,44 @@ var ERMrest = (function(module) {
 
             return module._markdownIt.render(value);
         },
+        
+        /**
+        * @function		    
+        * @param {Object} value A json value to transform
+        * @param {Object} [options] Configuration options.
+        * @return {string} A string representation of value
+        * @desc Formats a given json value into a string for display.
+        */
+        printJSON: function printJSON(value, options) {
```

Comment:
In the description of the function, comment that what it actually is returning based on different contexts.